component/async/sqs/sqs.go

```diff
@@ -11,15 +11,16 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/sqs"
 	"github.com/aws/aws-sdk-go/service/sqs/sqsiface"
+	"github.com/google/uuid"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/prometheus/client_golang/prometheus"
+
```

Comment:
Ordering seems funny, maybe a linter could help? Highly suggest using gofumpt

Reply:
You mean ordering of the imports? That's the goland standard one

Reply:
It looks like it's sorted alphabetically, isn't that the expected formatting?

Reply:
empty spaces